config/config.go

```diff
@@ -154,6 +155,26 @@ func LoadConfig(fname string) (*Config, error) {
 
 	log.Root().SetHandler(log.LvlFilterHandler(logLvl, log.Root().GetHandler()))
 
+	log.Error("this is the page size", "ps", cfg.PageSize)
+	if cfg.PageSize == 0 {
```

Comment:
Let's make this:

```
if cfg.PageSize < 4096 {
```

Because a page size of 2 is going to have serious problems.